cli/src/cmd/install.rs

```diff
@@ -105,13 +117,25 @@ fn install_as_folder(dep: &Dependency, libs: &Path) -> eyre::Result<()> {
 /// installs the dependency as new submodule
 fn install_as_submodule(dep: &Dependency, libs: &Path, no_commit: bool) -> eyre::Result<()> {
     // install the dep
-    Command::new("git")
+    let status = Command::new("git")
         .args(&["submodule", "add", &dep.url, &dep.name])
         .current_dir(&libs)
         .stdout(Stdio::piped())
         .stderr(Stdio::piped())
-        .spawn()?
-        .wait()?;
+        .status()?
+        .exit_ok()
```

Comment:
We cannot be using this due to the nightly req https://doc.rust-lang.org/std/process/struct.ExitStatus.html#method.exit_ok, could you do the check without the feature
?

Reply:
Okay sure, making the changes!